.github/workflows/release-embedding-sdk.yml

```diff
@@ -0,0 +1,173 @@
+name: Release Metabase Embedding SDK for React
+
+on:
+  workflow_dispatch:
```

Comment:
Do you plan on relying on workflow dispatch for the release, or is this temporarily for testing?

Reply:
Good question!

This is not temporary. We designed this the same way our [`uberjar.yml`](https://github.com/metabase/metabase/blob/214a59894e903783bb0bfb2f8759a276bc12167c/.github/workflows/uberjar.yml#L17) workflow works (it also accepts manual workflow trigger).

[Here's the discussion](https://metaboat.slack.com/archives/C063Q3F1HPF/p1715336732339349?thread_ts=1715336587.301399&cid=C063Q3F1HPF)